src/netbsd.rs

```diff
@@ -1,15 +1,31 @@
 //! Implementation for NetBSD
-use crate::{lazy::LazyPtr, util_libc::sys_fill_exact, Error};
-use core::{ffi::c_void, mem::MaybeUninit, ptr};
+use crate::{util_libc::sys_fill_exact, Error};
+use core::{
+    cmp,
+    ffi::c_void,
+    mem::{transmute, MaybeUninit},
+    ptr,
+    sync::atomic::{AtomicPtr, Ordering},
+};
+
+unsafe extern "C" fn polyfill_using_kern_arand(
+    buf: *mut u8,
+    buflen: libc::size_t,
+    flags: libc::c_uint,
+) -> libc::ssize_t {
+    debug_assert_eq!(flags, 0);
 
-fn kern_arnd(buf: &mut [MaybeUninit<u8>]) -> libc::ssize_t {
     static MIB: [libc::c_int; 2] = [libc::CTL_KERN, libc::KERN_ARND];
-    let mut len = buf.len();
+
+    // NetBSD will only return up to 256 bytes at a time, and
+    // older NetBSD kernels will fail on longer buffers.
+    let mut len = cmp::min(buflen, 256);
```

Comment:
Using `core::cmp::min` will allow us to remove the `cmp` import.

Reply:
I don't have strong preferences but I was trying to be consistent with what's done for the other submodules.

Reply:
I think this is fine as-is.